CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.36.0 (January 2, 2024)
+
+IMPROVEMENTS:
+* Remove hard coded version data in tests. [GH-1839](https://github.com/hashicorp/consul-template/pull/1839)
```

Comment:
I'd remove this as it is not really relevant to users

Reply:
See https://github.com/hashicorp/consul-template/pull/1862/commits/9e584cfb1f774300bb85776a0642ad189bd31d69